app/models/item.rb

```diff
@@ -7,6 +7,10 @@ class Item < ActiveRecord::Base
   has_many :orders, through: :order_items
   belongs_to :user
   belongs_to :event
+  has_attached_file :ticket
+  validates_attachment :ticket,
+    content_type: { content_type: "application/pdf" }
+
```

Comment:
does this just validate that the ticket is a PDF and not another format? Is there a way to validate that if the item is a delivery method that they have to attach a file?